client/Mac/cli/CMakeLists.txt

```diff
@@ -66,9 +67,19 @@ set_target_properties(${MODULE_NAME} PROPERTIES RESOURCE "${${MODULE_PREFIX}_RES
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -F../")
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -F../")
 
+if(CMAKE_GENERATOR MATCHES "Ninja")
```

Comment:
this looks like some duplication? you already defined that one directory above, so it should be defined here? (if not move the definition before the `add_subdirectory`)

Reply:
also, same notes as above apply

Reply:
I'd generally agree but as client/Mac and client/Mac/cli are created as separate cmake `project`s and both affected by the same issue I did copy it as a band aid fix. In general the MacOS client building could use "some" improvements..